draft-ietf-privacypass-http-api.md

```diff
@@ -329,9 +329,14 @@ endpoint, which would perform its own redemption {{token-redemption}},
 or the client redeeming the token and passing the result along to the
 target endpoint. These two methods are described below.
 
+In the HTTP ecosystem, redemption contexts should generally be keyed
+by the same privacy boundary used for cookies and other local
+storage. Generally this is the top-level origin.
```

Comment:
Should we link back to the architecture doc here as well, just to reinforce the connection.
```suggestion
storage. Generally this is the top-level origin. Any redemption context should be built following the principles outlined in {{draft-ietf-privacypass-architecture}} and later in {{privacy}}.
```